bliss/encoder/metrics.py

```diff
@@ -178,6 +179,192 @@ def plot(self):
         return fig, axes
 
 
+class DetectionPerformanceOnFlux(DetectionPerformance):
```

Comment:
This class appears to be duplicating much of the code in `DetectionPerformance` (twice in fact -- once for stars and once for galaxies). That's probably not what we want. If our goal is to break out detection performance by source type, can we just add an argument to the constructor `DetectionPerformance` that filters by source type if the argument is not `None`? Then, in the hydra config file, we'd instantiate `DetectionPerformance` three times: once for all light source, once for stars, and once for galaxies.